src/main/assembly/dist/bin/easy-update-solr-index

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
-BINDIR=`dirname $0`
-HOMEDIR=`dirname $BINDIR`
+BINPATH=`readlink -f $0`
```

Comment:
When I try to build and execute on my mac I get
 `readlink: illegal option -- f`
My current habit for this round-trip test that covers the shell-script and .jar:
  `mvn clean install ; tar -zxvf target/*.tar.gz ; mv *-SNAPSHOT target ;  */*/bin/*x --help`
Except for the last part before --help this command line would work for most of our modules